### ISSUE TITLE
Run without SSL?

### ISSUE THREAD

**Issue by jakejellinek** [CanastaWiki/Canasta#370]
Is it possible to configure the env/compose files such that SSL is not required? I am wanting to run this stack on a VM behind HA Proxy, and my SSL certificate are handled by the proxy. At the moment things don't work because a request is being made to obtain an SSL certificate which is not going through the proxy, so the stack never starts up.

**Comment by jeffw16** [comment]
You could have the HAProxy talk directly to the Varnish container. Feel free to change the networking routing in the docker-compose.override.yml file. Keep in mind this isn't officially supported. For anyone else who comes across this question on Google/Bing, this probably isn't the right way to solve your own situation.

**Comment by jakejellinek** [comment]
i realise you don't support this, and nobody else should do it...I did get it to apparently work by editing the docker-compose.override.yml (commented out caddy section and added in a varnish port 80 section) - so it all appears to be fine but I did notice that the canasta docker image thingy still says "unhealthy" - I presume this is related to my changes?

`CONTAINER ID   IMAGE                                                  COMMAND                  CREATED      STATUS                  PORTS                                         NAMES
c92507516678   ghcr.io/canastawiki/canasta:latest                     "/run-apache.sh"         4 days ago   Up 4 days (unhealthy)   80/tcp `

The logs just keep saying this:-

`mwjobrunner waits for 2 seconds...
Job queue is empty.
Job queue is empty.
Job queue is empty.
Job queue is empty.
mwtranscoder waits for 60 seconds...
Job queue is empty.
Job queue is empty.
Job queue is empty.
mwjobrunner waits for 2 seconds...
Job queue is empty.
`
